website/client/js/services/userServices.js

```diff
@@ -633,7 +633,7 @@ angular.module('habitrpg')
           var isStaticOrSocial = $window.location.pathname.match(/^\/(static|social)/);
           if (!isStaticOrSocial){
             localStorage.clear();
-            window.location.href = '/logout';
+            $window.location.href = '/logout';
```

Comment:
eh okay but this way I have to check every test file to find if it's happening or not... it takes ages